### PR TITLE
Complete all tasks

### PR DESCRIPTION
**Pull request by pottsnoc**

### CODE REVIEW

File: Exercises/1-remove.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 const removeElement = (array, item) => {
-  // Remove item from array modifying original array
+  const idx = array.findIndex(el => el === item);
```
Comment (tshemsedinov) [inline Exercises/1-remove.js]: Use `indexOf` here

File: Exercises/2-elements.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 'use strict';
 
 const removeElements = (array, ...items) => {
-  // Remove multiple items from array modifying original array
+  items.forEach(item => {
```
Comment (tshemsedinov) [inline Exercises/2-elements.js]: Use single loop for..of instead of nested forEach and findIndex

File: Exercises/3-unique.js [hunk 3]
```diff
@@ -3,6 +3,19 @@
 // Create and return a new array without duplicate elements
 // Don't modify initial array
 
-const unique = array => [];
+const unique = array => {
+  const table = {};
```
Comment (tshemsedinov) [inline Exercises/3-unique.js]: We do not need this auxiliary structure here

File: Exercises/3-unique.js [hunk 4]
```diff
@@ -3,6 +3,19 @@
...
+  const res = [];
+  array.forEach(el => {
```
Comment (tshemsedinov) [inline Exercises/3-unique.js]: Use for..of

File: Exercises/4-difference.js [hunk 5]
```diff
@@ -3,6 +3,8 @@
 // Find difference of two arrays
 // elements from array1 that are not includes in array2
 
-const difference = (array1, array2) => [];
+const difference = (array1, array2) =>
+  // Solution is not short
+  array1.filter(element => !array2.includes(element));
```
Comment (tshemsedinov) [inline Exercises/4-difference.js:8]: Looks good to me

**Review by tshemsedinov — CHANGES_REQUESTED** [review]

File: Exercises/3-unique.js (hunk 3 above)
Comment (pottsnoc) [reply Exercises/3-unique.js]: I thought that search in hash table works faster than search in array

**Comment by tshemsedinov** [comment]
:+1:

**Pull request closed without merging**